### DISCUSSION TITLE
Settings, Reverb, warning that can savely be ignored

### DISCUSSION THREAD

**Discussion by willeke1234** [GrandOrgue/grandorgue#1050] [General]
When the Reverb setting is switched on, a dialog appears "Warning, This feature is currently not supported.". The Help says: "A pop-up is displayed which can be safely ignored.". The Reverb is switched on and is working. Why are we getting this warning?

**Comment by wpostma** [comment]
That seems like it must be a bug. I also get it, perhaps I could figure out how to make it not happen.

While I'm here mentioning things I wish that the reverb level had a slider in the main Grand Orge panels or the main Grande Orge window, rather than a numeric input from 0 to 1.0 which seems like a silly way to set the reverb level.

It might be nice if the program shipped with a default .wav file and was ready to do convolution reverb out of the box also. I am sure there are people who would rather it have any IR wav at all.
